worker/games.py

```diff
@@ -1372,16 +1372,17 @@ def parse_options(s):
     except WorkerException as e:
         raise e
 
-    try:
-        verify_signature(
-            new_engine,
-            run["args"]["new_signature"],
-            games_concurrency * threads,
-        )
-    except RunException as e:
-        run_errors.append(str(e))
-    except WorkerException as e:
-        raise e
+    if run["args"]["base_signature"] != run["args"]["new_signature"]:
+        try:
```

Comment:
Are you sure this is ok? What if the author of the test supplies a bogus "new_signature" equal to "base_signature"? Are these signatures validated elsewhere?

Reply:
sorry I meant `new_engine != base_engine` but I your point holds nevertheless

Reply:
well we can check both

Reply:
Yes `new_engine != base_engine` seems to be a better test!